_data/projects/learn-git.yml

```
@@ -0,0 +1,12 @@
+name: Learn Git - First Contributions
+desc: This is perhaps one of the easiest ways to get introduced to open source
+site: https://github.com/rcallaby/Learn-Git/blob/main/First-Contributions/firstcontribution.md
+tags:
+  - learngit
+  - opensource
+  - firstcontribution

```

:art: given these are in the title, let's drop them from also being in the tags

```suggestion
  - open-source
```